mobile/src/features/Swap/useCases/ListOrders/ListOrders.tsx

```diff
@@ -51,8 +53,25 @@ export const ListOrders = () => {
     onBack: navigateToTxHistory,
   })
 
+  const isTestnet = network !== Chain.Network.Mainnet
```

Comment:
network manager has a isMainnet or isTestnet property if Im not mistaken

Reply:
checked properly and it is not really needed. thank guys